### ISSUE TITLE
Can it generate sample XML based on XSD specification?

### ISSUE THREAD

**Issue by OSA413** [redhat-developer/vscode-xml#907]
I was looking for ways of generating XML files for tests based on their XSD specification. I found several IDEs and this extension. The ability to generate such files would be a good feature.

**Comment by angelozerr** [comment]
No, but it is the same issue than https://github.com/redhat-developer/vscode-xml/issues/598
